### PR TITLE
Right-align state space table

### PR DESCRIPTION
**Pull request by philipmw**
This table is all integers (and a date)

### CODE REVIEW

File: org.lamport.tla.toolbox.tool.tlc.ui/src/org/lamport/tla/toolbox/tool/tlc/ui/editor/page/ResultPage.java [hunk 1]
```diff
@@ -1027,6 +1027,7 @@ public static void createTableColumns(Table stateTable, ResultPage page)
             for (int i = 0; i < columnTitles.length; i++)
             {
                 TableColumn column = new TableColumn(stateTable, SWT.NULL);
+                column.setAlignment(SWT.RIGHT);
```
Comment (lemmy) [inline org.lamport.tla.toolbox.tool.tlc.ui/src/org/lamport/tla/toolbox/tool/tlc/ui/editor/page/ResultPage.java]: @quaeler Does this alignment conflict with your recent UI styling?

Comment (quaeler) [reply org.lamport.tla.toolbox.tool.tlc.ui/src/org/lamport/tla/toolbox/tool/tlc/ui/editor/page/ResultPage.java]: Nope - it should not.

**Comment by lemmy** [comment]
Will look at this once https://github.com/quaeler/tlaplus/commit/3ef0532a48e1dc6446d7f401ee05b25d87f1cf47 has been merged.

**Comment by lemmy** [comment]
@philipmw Will take a look at this once we are done with a major Model Editor overhaul.

**Comment by lemmy** [comment]
@philipmw Thanks, but lets revisit after the model editor has received its overhaul.

**Pull request closed without merging**